compiler/rustc_lint/src/early.rs

```diff
@@ -313,7 +350,7 @@ pub trait EarlyCheckNode<'a>: Copy {
     fn attrs<'b>(self) -> &'b [ast::Attribute]
     where
         'a: 'b;
-    fn check<'b, T: EarlyLintPass>(self, cx: &mut EarlyContextAndPass<'b, T>)
+    fn check<'b, 'c, T: EarlyLintPass>(self, cx: &mut EarlyContextAndPass<'b, 'c, T>)
```

Comment:
Question: sorry, where did `'c` come from...? Or is this `'tcx`?

Reply:
It's `'tcx` ;-)